solid-terms.ttl

```diff
@@ -1,212 +1,283 @@
-@base <http://www.w3.org/ns/solid/terms> .
-@prefix : <#> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
-@prefix owl: <http://www.w3.org/2002/07/owl#> .
-@prefix cc: <https://creativecommons.org/ns#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix log: <http://www.w3.org/2000/10/swap/log#> .
-@prefix ldp: <http://www.w3.org/ns/ldp#> .
-
-<>
+# See details within this document for linkage to specification and purpose.
+# This ontology file is a non-normative supporting document.
+
+# Solid terms ontology
+
+## Ontology preferred namespace prefix and URI
+prefix solid: <http://www.w3.org/ns/solid/terms#>
+
+## Other namespaces used
+### Core W3C ontologies (OWL 2 reserved vocabulary)
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+### Other ontologies
+prefix dc: <http://purl.org/dc/terms/>
+prefix ldp: <http://www.w3.org/ns/ldp#>
+prefix log: <http://www.w3.org/2000/10/swap/log#>
+prefix vann: <http://purl.org/vocab/vann/>
+
+
+## Ontology
```

Comment:
```suggestion
prefix solid: <http://www.w3.org/ns/solid/terms#>
prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
prefix owl: <http://www.w3.org/2002/07/owl#>
prefix xsd: <http://www.w3.org/2001/XMLSchema#>
prefix dc: <http://purl.org/dc/terms/>
prefix ldp: <http://www.w3.org/ns/ldp#>
prefix log: <http://www.w3.org/2000/10/swap/log#>
prefix vann: <http://purl.org/vocab/vann/>

```

I don't think these comments are necessary or help.

I have to say that I much prefer the `@` pattern with terminating `.` for the sake of consistent look/feel... but not going to fight for it. Not sure if reuse for SPARQL is a strong argument to change either..

Reply:
Mmmh, interesting. I am very sensitive to the argument of consistency.
The way I see it, if SPARQL syntax is supported, then using it by default lowers the cognitive load, especially for newcomers.
I would like to see it becoming the new default for htat reason. It will take a while to reach widespread use but I think in the long term it's worth it.